MementoMori/MementoMori.Server/Service/DeckHelper.cs

```diff
@@ -86,6 +85,31 @@ public void UpdateDeck(EditedDeckDTO editedDeckDTO, Guid requesterId)
                 throw;
             }
         }
+        public void CreateDeck (DeckEditableProperties deck, Guid requesterId)
+        {
+            Deck newDeck = new()
+            {
+                Id = deck.Id,
+                CreatorId = requesterId,
+                isPublic = deck.isPublic,
+                Title = deck.Title,
+                Description = deck.Title,
+                Tags = deck.Tags,
+                Rating = 0,
+                RatingCount = 0,
+                Modified = DateOnly.FromDateTime(DateTime.Now),
+                Cards = null,
+                CardCount = 0,
+            };
+            _context.Decks.Add(newDeck);
```

Comment:
Should probably set all properties of the new deck here instead calling `UpdateDeck` right afterwards.